cpp/CMakeLists.txt

```diff
@@ -430,7 +413,7 @@ target_include_directories(cugraph
 # - link libraries --------------------------------------------------------------------------------
 
 target_link_libraries(cugraph PRIVATE
-    gunrock cublas cusparse curand cusolver cudart cuda ${LIBCYPHERPARSER_LIBRARY} ${MPI_CXX_LIBRARIES} ${NCCL_LIBRARIES})
+    gunrock cublas cusparse curand cusolver cudart cuda ${MPI_CXX_LIBRARIES} ${NCCL_LIBRARIES})
```

Comment:
unrelated but ${MPI_CXX_LIBRARIES} caught my attention, not sure it is really needed here

Reply:
dropped MPI_CXX_LIBRARIES